analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.cs

```diff
@@ -96,51 +100,61 @@ public void ThrowInLoop(object o)
             }
         }
 
+        public void ConstField(bool a, bool b)
+        {
+            var x = t || a || b; // Compliant t is const
+
+            if (t == true) // Noncompliant
```

Comment:
This should be indeed noncompliant as we are evaluating the whole condition as a whole `t == true`. 
Am I missing something?

Reply:
FN, good catch!